### PR TITLE
Ensure all gems for SQL Server app setup.

### PR DESCRIPTION
**Pull request by metaskills**
This pull request is related to #27820 which ensures that new applications setup with SQL Server have all conventions in place. However, when working today on the [SQL Server - Getting Started](https://www.microsoft.com/en-us/sql-server/developer-get-started/) pages I found out that the TinyTDS gem was not included in the Gemfile when generating a new application. This gem is the low level `raw_connection` for the adapter for all platforms of the SQL Server adapter. 

I will also be making a pull request to 5-0-stable as well. Thanks in advance!

### CODE REVIEW

File: railties/lib/rails/generators/app_base.rb
Comment (maclover7) [inline railties/lib/rails/generators/app_base.rb]: If possible, can we have this on the same line as the other `sqlserver` dependency specified on line 286? Or at least on the line above / below?

**Review by maclover7 — CHANGES_REQUESTED** [review]

**Comment by maclover7** [comment]
@metaskills Core / committers can backport for you once the PR is merged to master :)

File: railties/lib/rails/generators/app_base.rb
Comment (metaskills) [inline railties/lib/rails/generators/app_base.rb]: Oh yea, I totally missed that! Is it even possible to add two gems?

Comment (rafaelfranca) [inline railties/lib/rails/generators/app_base.rb]: hmmm, not right now, although it should be. Why `tiny_tds` is not a dependency of `activerecord-sqlserver-adapter`?

Comment (metaskills) [inline railties/lib/rails/generators/app_base.rb]: Because the `tiny_tds` gem is equal to `pg`. It is the low level `raw_connection`.

Comment (metaskills) [inline railties/lib/rails/generators/app_base.rb]: Oh.. oh oh... it should be!

**Comment by metaskills** [comment]
I'm gonna close this out... back in the day the SQL Server adapter had two connection modes. Now that is not the case, we can make it a true run time dependency of the adapter and put the concern in the right place. Thanks y'all!

**Comment by rafaelfranca** [comment]
👍

**Comment by maclover7** [comment]
👍

**Pull request closed without merging**
